Common/include/containers/CLookUpTable.hpp

```diff
@@ -284,6 +295,27 @@ class CLookUpTable {
                                                               const su2double val_CV1, const su2double val_CV2,
                                                               const su2double val_CV3);
 
+  /*!
+   * \brief Find the triangle within which the query point (val_CV1, val_CV2) is located.
+   * \param[in] val_CV1 - First controlling variable value.
+   * \param[in] val_CV2 - Second controlling variable value.
+   * \param[in] id_triangle - Reference to inclusion triangle index.
+   * \param[in] iLevel - Table level index.
+   * \returns 0 if inside data set, 1 if outside.
+   */
+  unsigned long FindInclusionTriangle(const su2double val_CV1, const su2double val_CV2, unsigned long& id_triangle,
```

Comment:
Return value can be a boolean?  (If yes, the maybe change behavior to: true=inside, false=outside)

```suggestion
   * \returns 0 if inside data set, 1 if outside.
   */
  bool FindInclusionTriangle(const su2double val_CV1, const su2double val_CV2, unsigned long& id_triangle,
```

Reply:
I changed the query inclusion flag of all lookup operations now to booleans instead of unsigned.